docs/3-Forms/switch.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Switch 
+---
+
+## On this page we will go through our Switch component 
+
+Switches allow the user to select one or more options from a set of items.
+They can be used in forms, filters, lists etc.
+
+### Table of contents
+
+<div class="row">
+    <div class="col-4">
+        <ul class="document__unordered-list">
+             <li class="document__unordered-list-item">
+                 <a class="msds-link" href="#overview">Overview</a>
+             </li>   
+              <li class="document__unordered-list-item">
+                 <a class="msds-link" href="#states">States</a>
+             </li>          
+             <ul class="document__unordered-list">
+                <li class="document__unordered-list-item">
+                    <a class="msds-link" href="#checked-state">Checked State</a>
+                </li>
+                <li class="document__unordered-list-item">
+                    <a class="msds-link" href="#disabled-state">Disabled State</a>
+                </li>
+                <li class="document__unordered-list-item">
+                    <a class="msds-link" href="#readonly-state">Readonly State</a>
+                </li>
+            </ul>             
+         </ul>    
+    </div>    
+</div>
+
+## Overview
+Each Switch component is based on a global CSS class called "msds-Switch". It is important to include it first as it is the main CSS class. Without it, the UI of the component won't work.
```

Comment:
Minor thing, here should be "msds-switch", by mistake you wrote a capital S.

Reply:
aww you got me XD